modules/sde-external-services/portal/src/main/java/org/eclipse/tractusx/sde/portal/model/response/KeycloakTokenResponse.java

```diff
@@ -0,0 +1,38 @@
+
```

Comment:
you might wanna consider some basic java linting tool to enforce a copyright header and a base line of code formatting.

You also have a little bit of a mixup between tabs and spaces.

Reply:
Tab spaces and lint issue resolved.